addons/website_sale_stock/models/product_template.py

```diff
@@ -48,11 +48,12 @@ def _get_additionnal_combination_info(self, product_or_template, quantity, date,
                     'product_with_stock_notification_enabled', set())
             )
             stock_notification_email = request and request.session.get('stock_notification_email', '')
+            precision_digits = self.env['decimal.precision'].precision_get('Product Unit of Measure')
             res.update({
                 'free_qty': free_qty,
                 'cart_qty': product_sudo._get_cart_qty(website),
                 'uom_name': product_sudo.uom_id.name,
-                'uom_rounding': product_sudo.uom_id.rounding,
+                'uom_rounding': 10 ** -precision_digits,
```

Comment:
The only time `uom_rounding` seems to be used is to determine how many digits to show through a log10. Could spare some efforts and write the precision digits directly